### PR TITLE
Cleaned up native ios code style, remove 3dsecure, convert index.ios.js to typescript

### PR DESCRIPTION
**Pull request by millibeckers**
Hide whitespace changes definitely recommended for this PR

Tested by quickly dragging the files into our project and compiling and running one of the paypal methods. 

TSConfig file copied from our repo with minimal changes

### CODE REVIEW

File: ios/RCTBraintree/RCTBraintree.h [hunk 1]
```diff
@@ -11,24 +11,27 @@
...
 
 + (instancetype)sharedInstance;
+
 - (BOOL)application:(UIApplication *)application
             openURL:(NSURL *)url
-            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *) options;
-- (void) run3DSecureCheck: (NSDictionary *)parameters callback: (RCTResponseSenderBlock)callback;
+            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options;
+
+- (void)run3DSecureCheck:(NSDictionary *)parameters
```
Comment (TGier) [inline ios/RCTBraintree/RCTBraintree.h]: i thought you removed 3D Secure?

File: index.ios.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          });
+        } else {
+          reject(err);
+        }
+      });
+    });
+  }
+
+  // 3DSecure not implemented or needed
+  check3DSecure(): Promise<boolean> {
```
Comment (TGier) [inline index.ios.ts]: do we call this anywhere? if not, why keep it around?

File: ios/RCTBraintree/RCTBraintree.m [hunk 3]
```diff
@@ -29,274 +32,243 @@ + (instancetype)sharedInstance {
...
-                    }
-                }else{
-                    // 3D Secure authentication was not possible
-                    args = @[[NSNull null], parameters[@"nonce"]];
-                }
-                callback(args);
-            }];
-    
+// 3DSecure removed - we are not including the 3DSecure functionality
+- (void)run3DSecureCheck:(NSDictionary *)parameters
```
Comment (TGier) [inline ios/RCTBraintree/RCTBraintree.m]: If we aren't including it, we shouldn't include it.

**Comment by TGier** [comment]
👍 
Also, small note on the force commit stuff/commit amending. I tend to use the "changes since last viewed" when requesting changes since it makes it easier to find specifics with a large review like this, but making it one commit again instead of just squashing it removes the convenience feature.

**Review by pdrlima — APPROVED** [review]
:shipit:

**Pull request merged**